lua/core/cmp.lua

```diff
@@ -73,10 +73,10 @@ M.config = function()
       { name = "crates" },
     },
     mapping = {
-      ["<C-d>"] = cmp.mapping.scroll_docs(-4),
-      ["<C-f>"] = cmp.mapping.scroll_docs(4),
+      ["<C-D>"] = cmp.mapping.scroll_docs(-4),
```

Comment:
I understand mapping tab and space but `C-d` is a completely different mapping than `C-D`.  I'm not sure the letter mappings need to be uppercased.

Reply:
You are right. It is my mistake.